languages/elixir/exercises/concept/lists/.docs/introduction.md

```diff
@@ -0,0 +1,36 @@
+Lists are built-in to the Elixir language. They are considered a basic type, denoted by square brackets. Lists may be empty or hold any number of items. A list's contents are not required to have the same type:
+
+```elixir
+empty_list = []
+one_item_list = [1]
+two_item_list = [1, 2]
+multiple_type_list = [1, :pi, 3.14]
+```
+
+Elixir implements lists as a linked list, where each node stores the reference to the next list. The first item in the list is referred to as the _head_ and the remaining list of items is called the _tail_. We can use this notation in code:
```

Comment:
```suggestion
Elixir implements lists as linked lists, where each node stores the reference to the next item in the list. The first item in the list is referred to as the _head_ and the remaining list of items is called the _tail_. We can use this notation in code:
```

Reply:
Conceptually, the introduction of lists as a list of items is probably easier and more similar to arrays in other languages.  If we want to point out that lists in Elixir are in fact hierarchical recursive data structures I think it might merit a bit more explanation.  The first time I realized that `[1, 2, 3]` really was something more like `[1, [2, [3]]]` it was a head trip.  The book "Elixir in Action" by Sasā Jurić has a good explanation of this, maybe we can dip into it?

Reply:
dip into it as another exercise? or a more detailed explanation?

Maybe there is a way to way to describe it at a high enough level in the introduction to get through the exercise, then explain more indepth in the after.md?